utils/apiUtil.js

```diff
@@ -113,6 +113,18 @@ const gen3Res = function (_chai, _) {
 };
 
 module.exports = {
+  /**
+   * Returns the JSON for an access token header given the token itself
+   * @param {string} accessToken - access token string
+   * @returns {JSON}
+   */
+  getAccessTokenHeader(accessToken) {
```

Comment:
awesome!! by searching "`bearer" in the repo i find a bunch of places we could use this :p if you feel like refactoring stuff :p

Reply:
:+1: